File: Tarea10.java
/* Nombre: JOSE PABLO GONZALEZ FLORES 
   Matr: 1986677 
   Hora: V5
 */
import java.util.Scanner;
public class Tarea10 {
    public static void main (String[] args){
               Scanner lec = new Scanner(System.in);
        int N;
         System.out.print("Ingresa el numero entero: ");
        N = lec.nextInt();
        for (int I=1; I<=N; I++){
          System.out.println(Math.pow(I,((2*I)-1)));  
        }
    
    }//Fin main

}//Fin de la clase

File: Tarea10.rb
=begin
Nombre:Jose Pablo González Flores Matr.:1986677 Hora:V5
=end
print("Ingresa el total de elementos del vector: ")
n = gets().to_i
puts()
puts()
print("A=[")
for i in (1..n).step 1
  res=i**((2*i)-1)
print("#{res},")
end
puts("]")
